client/my-sites/checkout/upsell-nudge/index.jsx

```diff
@@ -132,13 +136,13 @@ export class UpsellNudge extends React.Component {
 					value: postalCode,
 					isTouched: true,
 					errors: [],
-					isRequired: true,
+					isRequired: false,
```

Comment:
Note to self: will the validation endpoint return an error if the postal code is required but empty without this additional client-side check?

Reply:
ok, I'm pretty sure this is safe because the validation on the validation endpoint uses the same code as the validation on the transactions endpoint, so even if it (erroneously?) allows an empty postal code, the user's experience should be fine and they should not be aware.